rest/handler.go

```diff
@@ -1033,6 +1044,30 @@ func (h *handler) checkPublicAuth(dbCtx *db.DatabaseContext) (err error) {
 	return nil
 }
 
+// verifyWebsocketToken checks for a valid websocket token in the request headers. If present, invalidate the token so it can't be reused.
+//
+// Returns a nil error if there is no token on the header but no user object. If the header is found,
+func (h *handler) getWebsocketToken() string {
+	// go blip expects only one Sec-WebSocket-Protocol header, with comma separated values
+	protocolHeaders := h.rq.Header.Get(secWebSocketProtocolHeader)
+	var outputHeaders []string
+	var sessionID string
+	for _, header := range strings.Split(protocolHeaders, ",") {
+		header := strings.TrimSpace(header)
```

Comment:
The TrimSpace here means that outputHeaders might not exactly match the incoming headers.  I don't see how this would cause a problem, but it would be preferable to avoid changing anything other than the blipSessionIDPrefix header here?